Content.Server/Botany/Systems/PlantHolderSystem.cs

```diff
@@ -268,16 +262,13 @@ private void OnInteractUsing(Entity<PlantHolderComponent> entity, ref InteractUs
             var displayName = Loc.GetString(component.Seed.DisplayName);
             _popup.PopupCursor(Loc.GetString("plant-holder-component-take-sample-message",
                 ("seedName", displayName)), args.User);
-            component.Health -= (_random.Next(3, 5) * 10);
+            component.Health -= _random.Next(30, 50);
```

Comment:
These are not equivalent. The former returns only 30, 40, or 50. This patch returns all of the intermediate values. Why do you want to do this?

Reply:
already mentioned in pr description, it makes it less predictable